src/app/pages/authentication/signup/signup.page.html

```diff
@@ -11,7 +11,7 @@
 <ion-content class="auth-form">
   <ion-grid>
     <ion-row>
-      <ion-col align-self-center>
+      <ion-col class="align-self-center">
```

Comment:
Does this get the `ion-align-self-center` treatment? I don't know styling.

Reply:
It does. Missed this one too.